docs/reference/migration/migrate_6_0/mapping.asciidoc

```diff
@@ -0,0 +1,15 @@
+[[breaking_60_mapping_changes]]
+=== Mapping changes
+
+=== Analysis
+
+==== Synonym Token Filter
+
+In 6.0, Synonym Token Filter tokenize synonyms with whatever
+tokenizer and token filters appear before it in the chain.
+
+`tokenizer` and `ignore_case` are deprecated.
+These parameters are still left for backwards compatibility.
+If `tokenizer` is set, this token filter will have the same behavior as 5.x.
```

Comment:
This does not look true based on the code? We seem to ignore it for new indices all the time (which is fine)?